doc/citing.rst

```diff
@@ -3,23 +3,25 @@
 Citing Matplotlib
 =================
 
-If Matplotlib contributes to a project that leads to a scientific
-publication, please acknowledge this fact by citing `Hunter et al (2007)
-<https://doi.org/10.1109/MCSE.2007.55>`_ using this ready-made BibTeX entry:
+If Matplotlib contributes to a project that leads to a scientific publication,
+please acknowledge this fact by citing `J. D. Hunter, "Matplotlib: A 2D
+Graphics Environment", Computing in Science & Engineering, vol. 9, no. 3,
+pp. 90-95, 2007 <https://doi.org/10.1109/MCSE.2007.55>`_.
```

Comment:
(This is IEEE style, which is, well, consistent with the journal this was published in...)